camel/societies/workforce/workforce.py

```diff
@@ -4438,6 +4780,7 @@ def clone(self, with_memory: bool = False) -> 'Workforce':
             graceful_shutdown_timeout=self.graceful_shutdown_timeout,
             share_memory=self.share_memory,
             use_structured_output_handler=self.use_structured_output_handler,
+            max_refinement_iterations=2,
```

Comment:
```suggestion
            max_refinement_iterations=self.max_refinement_iterations,
```
I think it should be this instead of hardcoded value